integration-tests/tests/analysis-context-bar.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash/fp')
+const { withRegisteredUser, withBilling, withWorkspace, performAnalysisTabSetup } = require('../utils/integration-helpers')
+const {
+  click, clickable, getAnimatedDrawer, findElement, noSpinnersAfter
+} = require('../utils/integration-utils')
+
+
+const testAnalysisContextBarFn = _.flow(
+  withWorkspace,
+  withBilling,
+  withRegisteredUser
+)(async ({ page, token, testUrl, workspaceName }) => {
+  // Navigate to appropriate part of UI (the analysis tab)
+  await performAnalysisTabSetup(page, token, testUrl, workspaceName)
+  // Create a runtime
+  await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Environment Configuration' })) })
+  await findElement(page, getAnimatedDrawer('Cloud Environment Details'))
+  await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Settings' })) })
```

Comment:
It probably would be good to make sure that this is the Jupyter one. I guess it's working just because it is the first?